frontend/src/pages/MetadataByType/MetadataByTypeBody.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  padding: 0.5em;
+`;
+
+const Tag = styled.div`
+  background-color: #eeeeee;
+  color: #595959;
+  font-size: 0.9em;
+  padding: 0.1em 0.7em;
+  display: inline-block;
+  border-radius: 1em;
+  margin: 0.3em;
+`;
+
+const State = styled.div`
+  padding: 0 0.4em;
+  background-color: ${({ color }) => color};
+`;
+
+const releaseStates = [
+  <State color="#9999dd">
+    Released
+  </State>,
+  <State color="#00ff00">
+    Ready to release
+  </State>,
+  <State color="#ffff33">
+    Needs work
+  </State>,
+  <State color="#cc4444">
+    Not releasable
+  </State>,
+];
+
+export const MetadataByTypeBody = (props) => {
+  const { name } = props;
```

Comment:
Why not do the object destructuring inside the function parameter list as the destructuring only consists of one field.